tfx/orchestration/kubeflow/base_component.py

```diff
@@ -94,8 +93,6 @@ def __init__(self,
       metadata_ui_path: File location for metadata-ui-metadata.json file.
     """
 
-    utils.replace_placeholder(component)
```

Comment:
This was a noop given that it was changing component.exec_properties and it's not being used before. 

This caused me to not understand the failure mode, so removing it to make it clear.